README.md

```diff
@@ -155,19 +155,17 @@ If this looks strange to you, you can also use the more traditional [blocking AP
 As stated above, this library provides you a powerful, async API by default.
 
 If, however, you want to integrate this into your traditional, blocking environment,
-you should look into also using [clue/reactphp-block](https://github.com/clue/reactphp-block).
+you should look into also using [reactphp/async](https://github.com/reactphp/async).
 
 The resulting blocking code could look something like this:
 
 ```php
-use Clue\React\Block;
-
```

Comment:
Can you keep the use statement as we did in [clue/reactphp-mq#34](https://github.com/clue/reactphp-mq/pull/34/files).